packages/dashboard/src/lib/components/modals/TranslationModal.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+    // TODO: Implement actual machine translation
+
+    import {Button, ButtonSet, Modal, Tab, TabContent, Tabs, TextArea} from "carbon-components-svelte";
+
+    export let open = false;
+    export let machineTranslation = true; // This should be local settings
+    // Default values for understanding, will be removed once implemented
+    export let original = "Settings";
+    export let translations = {de: "Einstillinger", da: "Indstillinger"}; // Sounds more fun than einstellungen
+    export let newTranslations = false; // Will be true when user submits new translations
+
+    let originalTranslations = {...translations};
+</script>
+
+<Modal
+    bind:open
+    modalHeading="Translation Card"
+    size="sm"
+    hasform
+    primaryButtonText="Confirm translations"
+    secondaryButtonText="Cancel"
+    on:click:button--secondary={() => (open = false)}
+    on:submit={() => {open = false; newTranslations = true}}
```

Comment:
Why does the submission differ between new and "old" translation?

Reply:
Just an explicit way of saying "please update" when calling the component after it closes.